snapcraft/internal/lxd/_project.py

```diff
@@ -151,6 +155,30 @@ def _remote_mount(self, destination, source):
             'You can run the following command to enable it:\n'
             'echo Y | sudo tee /sys/module/fuse/parameters/userns_mounts')
 
+    def _setup_user(self):
+        # Setup user mirroring host user with sudo access
+        self._user = os.environ['USER']
+        self._project_folder = '/home/{}/build_{}'.format(
+            self._user, self._metadata['name'])
+        logger.debug('Setting up user {!r} in container'.format(self._user))
+        try:
+            self._container_run([
+                'useradd', self._user, '--create-home'])
+        except ContainerRunError as e:
+            # username already in use
+            if e.exit_code != 9:
```

Comment:
so, exit_code == 9 means username already in use and we ignore it?
The comment could be clearer, saying this, that the error will be ignored.